### PR TITLE
Jcs/pull tests done

### PR DESCRIPTION
**Pull request by jSylvestre**

### CODE REVIEW

File: Anlab.Core/Domain/Order.cs [hunk 1]
```diff
@@ -38,6 +38,8 @@ public class Order :IDatedEntity
 
         public string Status { get; set; }
 
+        public string OrderRequest { get; set; }
```
Comment (srkirkland) [inline Anlab.Core/Domain/Order.cs]: @jSylvestre is this like request number?

File: Anlab.Mvc/Controllers/HomeController.cs [hunk 2]
```diff
@@ -1,9 +1,19 @@
...
+//using AnlabMvc.Services;
+using Microsoft.AspNetCore.Mvc;
 
 namespace AnlabMvc.Controllers
 {
     public class HomeController : ApplicationController
     {
+        //private readonly ILabworksService _labworksService;
+
+        //public HomeController(ILabworksService labworksService)
```
Comment (srkirkland) [inline Anlab.Mvc/Controllers/HomeController.cs]: @jSylvestre dump

File: Anlab.Mvc/Services/DbInitializationService.cs [hunk 3]
```diff
@@ -49,6 +49,33 @@ public DbInitializationService(ApplicationDbContext context, UserManager<User> u
                 "AL-KCL",
                 "AL-OX",
                 "AL-PY",
+                "-BCL-P-IC",
```
Comment (srkirkland) [inline Anlab.Mvc/Services/DbInitializationService.cs]: @jSylvestre i know this is just test data so w/e, but I think the ones that start with dashes are not valid codes, they are like for QC or tests or something

File: Anlab.Mvc/Services/IOrderService.cs [hunk 4]
```diff
@@ -121,6 +123,43 @@ public void PopulateOrderWithLabDetails(OrderSaveModel model, Order orderToUpdat
...
+            var tests = await PopulateSelectedTestsItemModel(testIds);
+
+            var orderDetails = orderToUpdate.GetOrderDetails();
+            var calcualtedTests = new List<TestDetails>();
+            foreach (var test in tests)
+            {
+                var cost = orderDetails.Payment.IsInternalClient ? test.InternalCost : test.ExternalCost;
+                var costAndQuantity = cost * orderDetails.Quantity;
+
+                calcualtedTests.Add(new TestDetails
```
Comment (srkirkland) [inline Anlab.Mvc/Services/IOrderService.cs]: @jSylvestre this seems like a good method to extract and make public, i.e. create TestDetails given a test and an order details.  I think it comes up somewhere else too.

File: Anlab.Mvc/Views/Lab/UpdateFromCompletedTests.cshtml [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+</div>
+
+@section AdditionalStyles
+{
+    <link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/1.10.13/css/jquery.dataTables.min.css" />
+}
+
+@section Scripts
+{
+    <script type="text/javascript" src="https://cdn.datatables.net/1.10.13/js/jquery.dataTables.min.js"></script>
```
Comment (srkirkland) [inline Anlab.Mvc/Views/Lab/UpdateFromCompletedTests.cshtml]: @jSylvestre has nothing to do with this PR but it reminded me -- go ahead and create a set of partials for datatables, so you can do like `@Html.Partial("_DataTablesScripts")`.  Or if you have another idea that's good too, my goal is not to repeat the CDN path more than once, I'd hate to upgrade and have to change in each file.

File: Anlab.Core/Domain/Order.cs (hunk 1 above)
Comment (jSylvestre) [inline Anlab.Core/Domain/Order.cs]: @srkirkland Yes. I can rename it if you want.

File: Anlab.Mvc/Services/DbInitializationService.cs (hunk 3 above)
Comment (jSylvestre) [inline Anlab.Mvc/Services/DbInitializationService.cs]: @srkirkland Lets me see what comes back. For real data, we wouldn't include any of those in the DB so they would just get filtered out.

File: Anlab.Mvc/Services/IOrderService.cs (hunk 4 above)
Comment (jSylvestre) [inline Anlab.Mvc/Services/IOrderService.cs]: @srkirkland Yeah. There are enough differences that I want to think about it some more.

File: Anlab.Mvc/Views/Lab/UpdateFromCompletedTests.cshtml (hunk 5 above)
Comment (jSylvestre) [inline Anlab.Mvc/Views/Lab/UpdateFromCompletedTests.cshtml]: @srkirkland K. This was just a copy and past of the confirmation page with a couple minor changes.

**Pull request merged**